grip_width: 0.02400004
y: -0.2133337
x: 0.09028526